easybuild/easyconfigs/i/irrlicht/irrlicht-1.8.5-foss-2021b.eb

```diff
@@ -0,0 +1,62 @@
+easyblock = 'MakeCp'
+
+name = 'irrlicht'
+version = '1.8.5'
+
+homepage = 'https://irrlicht.sourceforge.io/'
+description = """
+The Irrlicht Engine is an open source realtime 3D engine written in C++. It is cross-platform, 
+using D3D, OpenGL and its own software renderers. OpenGL-ES2 and WebGL renderers are also in development.""" 
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+toolchainopts = {'usempi': False}
+
+source_urls = ['https://downloads.sourceforge.net/irrlicht']
+sources = ['%(name)s-%(version)s.zip']
+patches = [
+    ('Irrlicht-foss-2021b-makefiles.patch', 'source/Irrlicht'),
+]
+
+checksums = [
+    'effb7beed3985099ce2315a959c639b4973aac8210f61e354475a84105944f3d',  # irrlicht-1.8.5.zip
+    'bb285db76afd02a47abf5b5e6de668ee96eec3b022979acff49bab535aa3c343',  # Irrlicht-foss-2021b-makefiles.patch
+]
+
+builddependencies = [
+    ('UnZip', '6.0'),
+]
+
+dependencies = [
+    ('X11', '20210802'),
+    ('libjpeg-turbo', '2.0.6'),
+]
+
+skipsteps = ['configure']
+
+# Use appropriate makefile and flags for this toolchain
+prebuildopts = 'cd source/Irrlicht &&'
+buildopts = "sharedlib"
+
+files_to_copy = [
+    (['lib/Linux/libIrrlicht.so.%(version)s'], 'lib'),
+    (['bin/*'], 'bin'),
+    (['include/*'], 'include'),
+    (['source/*'], 'source'),
+]
+
+postinstallcmds = [
+    "cd %(installdir)s/lib"
+    "&& ln -s libIrrlicht.so.%(version)s libIrrlicht.so.1.8"
+    "&& ln -s -f libIrrlicht.so.1.8 libIrrlicht.so"
+    "&& cp libIrrlicht.so.%(version)s %(installdir)s"
+    "&& ln -s -f libIrrlicht.so.1.8.5 libIrrlicht.so.1.8"
+    "&& ln -s -f libIrrlicht.so.1.8 libIrrlicht.so"
+    "&& cd  %(installdir)s"
+]
+
+sanity_check_paths = {
+    'files': ['lib/libIrrlicht.so.%(version)s'],
```

Comment:
@absrocks It would be nice if we could check for one or more specific binaries here, and also add a sanity check command for them to run `--help` or `--version`.

We should also check for `lib/libIrrlicht.so`, to make sure the symlinking worked as expected (no broken symlink)